ui/src/components/SampleView/DrawGridPlugin.js

```diff
@@ -316,7 +316,13 @@ export default class DrawGridPlugin {
       result = gd.result[this.resultType];
     }
 
-    if (result !== undefined && result !== null && gd.id !== null) {
+    const result_length = Object.values(result).length;
+    if (
+      result !== undefined &&
+      result !== null &&
+      gd.id !== null &&
+      result_length > 0
```

Comment:
`Object.values()` throws when passed `undefined` or `null`.
```suggestion
    if (
      result !== undefined &&
      result !== null &&
      gd.id !== null &&
      Object.values(result).length > 0
```